src/software/jetson_nano/services/motor.h

```diff
@@ -286,6 +319,11 @@ class MotorService
     // Previous wheel velocities
     WheelSpace_t prev_wheel_velocities_;
 
+    int front_left_target_velocity  = 0;
+    int front_right_target_velocity = 0;
+    int back_left_target_velocity   = 0;
+    int back_right_target_velocity  = 0;
```

Comment:
nit: is the unit of these variables electrical RPM? If so, we should probably name it that instead of velocity

Reply:
fixed